src/content/security/index.md

```diff
@@ -0,0 +1,231 @@
+
+<DocLink to="/what-is-ethereum/" title="What is Ethereum?" />
+
+<DocLink to="/eth/" title="What is ether?" />
+<Divider />
+## Wallet Security
+
+### Don't give out your Private keys
+
+**Never, for anyone reason, hand out your private keys!**
+
+The private key to your wallet acts as your digital signature. It is the only thing stopping someone who knows your wallet address from draining your account of all of its assets!
+
+### Use a Hardware Wallet
+
+A hardware wallet provides offline storage for private keys. They are considered the most secure wallet option for storing your private keys.
+
+Keeping private keys offline massively reduces the risk of being hacked, even if a hacker gets control of your computer.
+
+#### Try a hardware wallet:
+
+- [Ledger](<[https://www.ledger.com/](https://www.ledger.com/)>)
+- [Trezor](<[https://trezor.io/](https://trezor.io/)>)
+
+### Double check transactions
+
+Accidently sending crypto to the wrong wallet address is a common mistake. Unless you know the address owner, there will be no way for you to retrieve your funds.
+
+Always make sure the address you are sending to exactly matches the desired recipient's address before sending a transaction.
+It is also recommended when interacting with a smart contract to read the transaction message before signing.
+### Set smart contract spend limits
+
+When interacting with smart contracts, do not allow unlimited spend limits. An unlimited spend could enable the smart contract to drain your wallet. Instead, set spending limits to only the amount necessary for the transaction.
+
+### Don't take screenshots of your seed phrases/private keys
+
+By screenshotting your seed phrases or private keys, you risk syncing them to the cloud and potentially making them accessible to hackers. Obtaining private keys from the cloud is a common attack vector for hackers.
+
+<Divider />
+
+## Common Scams
+
+Todo: add copy here
+
+### Giveaway scam (Double your ETH)
+
+One of the most common scams in cryptocurrency is the giveaway scam. The giveaway scam can take many forms, but the general premise is that if you send ETH to the provided wallet address, you will receive your ETH back but doubled. *For this reason, it is also known as the 2-for-1 scam.*
+
+These scams usually stipulate a limited time of opportunity to claim the giveaway to encourage poor decision-making and create a sense of false urgency.
+
+#### Twitter Hack
+
+A high-profile version of this occurred in July 2020, when the Twitter accounts of prominent celebrities and organisations got hacked. The hacker simultaneously posted a Bitcoin giveaway on the hacked accounts. Although the deceptive tweets were quickly noticed and deleted, the hackers still managed to get away with 11 bitcoin (or $500,000 as of September 2021)
+
+![A scam on Twitter](./twitterScam.png)
+
+#### Vitalik Giveaway
```

Comment:
Or any celebrity?

Reply:
Makes sense. 

I tried to pull some Algolia data for this but I couldn't search the range I wanted to check. My intuition here was that most people come to discord asking specifically about giveaways that Vitalik is doing and they might query search engines the same way. My hope was to capture that traffic. Could use your knowledge on how we could best do this from an SEO perspective.